docker/server/start.sh

```diff
@@ -6,4 +6,10 @@ then
     echo "${TURBINIA_CONF}" | base64 -d > /etc/turbinia/turbinia.conf
 fi
 
-/usr/local/bin/turbiniactl -S server
+# Use log file path from environment variable is it exists, else get the path from the config.
+if [ ! -z ${TURBINIA_LOG_FILE+x} ]
```

Comment:
What is the `+x` syntax used for here?  It looks similar to parameter expansion, but I don't see this exact syntax in the man page.   It seems to work fine, so I think it's ok, but I'm still curious for myself :).